### PR TITLE
Clean Up

### PR DESCRIPTION
**Pull request by opticod**
-  unused Drawables are removed(To make build faster)
- replaced  `e.printStackTrace();` with Timber Logging
- dependencies are upgraded 
- BaseActivity added in Manifest

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/dbutils/DatabaseOperations.java [hunk 1]
```diff
@@ -61,7 +63,7 @@
                 );
                 sessions.add(s);
             } catch (ParseException e) {
-                e.printStackTrace();
+                Timber.e("Parsing Error Occurred");
```
Comment (mananwason) [inline app/src/main/java/org/fossasia/openevent/dbutils/DatabaseOperations.java]: You should also add the cause of parsing error. or maybe just the identifier of the method where it occured.

Comment (opticod) [inline app/src/main/java/org/fossasia/openevent/dbutils/DatabaseOperations.java]: Ok,I will add the identifier of the method for more clear debugging purposes.

Comment (mananwason) [inline app/src/main/java/org/fossasia/openevent/dbutils/DatabaseOperations.java]: Great :+1:

**Comment by opticod** [comment]
@mananwason done with the changes.

**Comment by mananwason** [comment]
Looks good. Merging !!

**Comment by opticod** [comment]
:smile:

**Comment by creativepsyco** [comment]
This commit also removed a bunch of drawables and broke the build.

**Comment by mananwason** [comment]
I tested it on my machine. It was building fine :confused:

**Comment by mananwason** [comment]
Right now travis shows some issues. I'll add a fix for this by today.

**Comment by mananwason** [comment]
Right now travis shows some issues. I'll add a fix for this by today.

**Comment by creativepsyco** [comment]
because you probably compiled a version before that, you can try clean building it will fail. I have fixed this in commit eaa4893e8d4b624fe95e4c2b6d78cbb96b316b6f

**Comment by mananwason** [comment]
okay great. Maybe something changed after I tried building it.

**Comment by opticod** [comment]
I apologize for it.. @creativepsyco 
And regarding drawables..we have discussed in commit 2d2fb15 
Should i make a revert for those drawables?

**Comment by creativepsyco** [comment]
No need the build is okay now. It was just the launcher icon

**Pull request merged**